contents/blog/startup-data-stack.md

```diff
@@ -0,0 +1,108 @@
+- How do we best serve our [ideal customer](/blog/creating-ideal-customer-profile)?
+- How do we optimize our [conversion funnel](/tutorials/funnels)?
+- What channels should we [be advertising on](/blog/dev-marketing-for-startups)?
+
+To answer these questions, teams pull data from a growing number of sources like product analytics, CRM, help desk, and ad platforms. They also utilize more of the features of these sources, such as customer engagement and session recordings, generating more data.
+
+Bringing all this data together and managing it often requires a [customer data platform](/docs/integrate/cdp) (CDP). CDPs collect the data from different sources and send it for use in destinations. For example, it might collect product data from PostHog, advertising data from Google, and revenue data from Stripe, and send it back to those same tools, a warehouse, or a business intelligence tool like Hex.
+
+![Seed](../images/blog/startup-data-stack/seed.png)
+
+The goal of the stack at this stage is to give insights into how the product and specific features are doing. Teams want to answer questions about the product and get basic KPI reporting. This enables them to make decisions about what to work on and measure progress towards goals.
+
+Moving from individual sources to a CDP makes gathering data faster, but starts to create a web of integrations and mainteinance. It is a challenge to know if data is accurate, luckily at this stage, it doesn't need to be. The Seed stage strikes the balance of giving engineers enough data to know what to prioritize with having plenty of time to build it.
+
+## The Series A data stack
+
+Series A is where data begins to get serious, and by serious, I mean the startup makes someone responsible for owning the data stack. This person is often a head of data, data engineer, or backend engineer focused on data. They see their goal as empowering engineers with data to evaluate the success of what they build. To do this, they implement the so-called "modern data stack" which contains four tools:
+
+1. A data warehouse to store all the data and act as a single source of truth. Options include Snowflake, BigQuery, and RedShift.
+2. An ETL pipeline to extract data from different sources, transform it, and load it into the warehouse. Options include Fivetran, Integrate.io, and Airbyte.
+3. A data transformation tool to model data, clean it up, and make it usable. dbt is basically the only option.
+4. A business intelligence or visualization tool to get insights from the data you’ve collected. Options include Metabase, Looker, and Hex.
+![Series A](../images/blog/startup-data-stack/series-a.png)
+
+Startups need this stack because managing and accessing individual sources of data becomes unsustainable. Tools like analytics and the CDP don't give complete insight into performance. Teams need a single source of truth, and the modern data stack gives them that. This creates trust in the data and greater accuracy.
```

Comment:
```suggestion
Startups need this stack because managing and accessing individual sources of data becomes unsustainable. Tools like analytics and the CDP don't give complete insight into performance. Teams need a reliable, single source of truth they trust, and they hope the modern data stack will give them that.
```